examples/newrelicapm/otel-collector-config.yaml

```diff
@@ -37,7 +37,3 @@ service:
     metrics/apm:
       receivers: [newrelicapm/metrics]
       exporters: [otlp]
-    logs:
-      receivers: [otlp, newrelicapm]
```

Comment:
Do we need a pass-through section for logging, or will logging still work with this entire block removed?

Reply:
Yea, you're right, probably makes sense for our example to still demonstrate exporting logs. Also added the NR processor the logs pipeline... all it does is add `instrumentation.provider=newrelic-opentelemetry`.

Reply:
@alanwest we should also update the `README`, which contains an example config. Perhaps we could link to this config file instead of the current code block.